analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -466,6 +466,25 @@ def add_arguments_to_parser(parser):
                                    "same translation unit without "
                                    "Cross-TU enabled.")
 
+        # Only check for AST loading modes if CTU is available.
+        if analyzer_types.is_ctu_on_demand_available(context):
+            ctu_opts.add_argument('--ctu-ast-creation',
```

Comment:
```suggestion
            ctu_opts.add_argument('--ctu-ast-mode',
```
what do you think?

Reply:
This seems tricky to get right semantically. @martong @csordasmarton @bruntib  what do you think about `ctu-ast-mode`?

Reply:
I vote on the `ctu-ast-mode` name. It is more expressive.